data/TeXmacs/bin/tm_sympy

```diff
@@ -86,10 +86,10 @@ while True:
 
         try:
             output = eval(line, _globals)
-        except SyntaxError:
+        except:
```

Comment:
You should `except Exception`. That will allow `KeyboardInterrupt` to still stop the script (`EOFError` should probably exit the script too, unless that is a special character used by TeXmacs).

Reply:
That is not a problem for TeXmacs. You can always kill the process clicking on the scissors.